mmv1/products/oracledatabase/AutonomousDatabase.yaml

```diff
@@ -70,8 +72,9 @@ examples:
     ignore_read_extra:
       - 'deletion_protection'
     test_vars_overrides:
-      'project': '"oci-terraform-testing"'
-      'deletion_protection': 'false'
+      project: '"oci-terraform-testing"'
+      deletion_protection: 'false'
+      database_name: 'fmt.Sprintf("tftestdatabase%s", acctest.RandString(t, 10))'
```

Comment:
It looks like `database_name: 'mydatabase'` is also needed under `vars`.

Reply:
Oops, thanks for spotting that!